OSpackages.sh

```diff
@@ -199,6 +210,24 @@ function PrepChroot {
       ln -t "${CHROOTMNT}/etc" -s ./rc.d/init.d
    fi
 
+   # Satisfy weird, OL8-dependecy:
+   # * Ensure the /etc/dnf and /etc/yum contents are present
+   if [[ ${#DNF_ARRAY[*]:-} -gt 0 ]]
```

Comment:
This actually results in a "bad substitution" error. Sorry about that. This suggestion works though:

```suggestion
   if [[ -n "${DNF_ARRAY:-}" ]]
```